File: internal/discovery/testdata/fixtures/python_framework/graph.py
from langgraph.graph import StateGraph

def build():
    g = StateGraph(dict)
    g.add_node("classify", classify_fn)
    g.add_node("answer", answer_fn)
    g.add_node("escalate", escalate_fn)
    g.add_edge("classify", "route")
    g.add_conditional_edges("route", pick, {"faq": "answer", "esc": "escalate"})
    g.set_entry_point("classify")

File: internal/discovery/testdata/fixtures/typescript_framework/src/graph.ts
import { StateGraph } from "@langchain/langgraph";

const g = new StateGraph({ channels: {} });

g.addNode("classify", classifyFn);
g.addNode("answer", answerFn);
g.addNode("escalate", escalateFn);
g.addEdge("classify", "route");
g.addConditionalEdges("route", routeFn, { faq: "answer", esc: "escalate" });
g.setEntryPoint("classify");
